dirty_cat/minhash_encoder.py

```diff
@@ -181,42 +181,47 @@ def transform(self, X):
             template = ("handle_missing should be either 'error' or "
                         "'zero_impute', got %s")
             raise ValueError(template % self.handle_missing)
-        X_out = np.zeros((len(X), self.n_components))
-        X = X.reshape(-1)
-
-        # TODO Parallel run here
-        nan_idx = []
-
-        if self.hashing == 'fast':
-            for i, x in enumerate(X):
-                if isinstance(x, float): # true if x is a missing value
-                    nan_idx.append(i)
-                elif x not in self.hash_dict:
-                    X_out[i, :] = self.hash_dict[x] = self.get_fast_hash(x)
-                else:
-                    X_out[i, :] = self.hash_dict[x]
-
-        elif self.hashing == 'murmur':
-            for i, x in enumerate(X):
-                if isinstance(x, float):
-                    nan_idx.append(i)
-                elif x not in self.hash_dict:
-                    X_out[i, :] = self.hash_dict[x] = self.minhash(
-                        x,
-                        n_components=self.n_components,
-                        ngram_range=self.ngram_range
-                    )
-                else:
-                    X_out[i, :] = self.hash_dict[x]
-
-        else:
-            raise ValueError("hashing function must be 'fast' or"
-                             "'murmur', got '{}'"
-                             "".format(self.hashing))
-
-        if self.handle_missing == 'error' and nan_idx:
-            msg = ("Found missing values in input data; set "
-                   "handle_missing='' to encode with missing values")
-            raise ValueError(msg)
-
-        return X_out
+        X_enc = []
```

Comment:
The idea is the get rid of the "X_enc" as a list, and the "append" and "hstack" at the end